targets/runtime.project.json

```diff
@@ -7,26 +7,31 @@
   "frameworks": {
     "dnxcore50": {
       "dependencies": {
```

Comment:
@rainersigwald:

> I assume this necessitates a runtime.project.lock.json update as well?

It's updated in the same commit... or am I missing something